lib/src/models/device.dart

```diff
@@ -0,0 +1,51 @@
+part of spotify.models;
+
+@JsonSerializable()
```

Comment:
Please add `createToJson: false`, so we don't have unused code generated

```suggestion
@JsonSerializable('createToJson: false')
```

Reply:
Also please rebuild the json serializers after that change 😄 

```
pub run build_runner build
```